index.js

```diff
@@ -127,6 +136,13 @@ async function recursivelyMoveFilesToTempFolder() {
   }
 }
 
+async function getExistingComments() {
```

Comment:
How was this tested?

Reply:
I setup a GHA using `open-universities-australia/sfdx-scan-pull-request@main` on a private SF repo and ran multiple commits + re-running the workflow

Reply:
Sounds good - thank you for clarifying and testing this.

As an aside - I am working on a suite of Jest tests so we can run locally with mocked actions core, Octokit, etc. to make local testing much easier in the future. Github actions are a bit of a pain to test, so thank you again 🙏